src/test/java/org/broadinstitute/hellbender/utils/GATKProtectedVariantContextUtilsUnitTest.java

```diff
@@ -0,0 +1,58 @@
+package org.broadinstitute.hellbender.utils;
+
+import htsjdk.samtools.SAMFileHeader;
+import htsjdk.samtools.util.Locatable;
+import org.broadinstitute.hellbender.engine.ReadsContext;
+import org.broadinstitute.hellbender.engine.ReadsDataSource;
+import org.broadinstitute.hellbender.utils.pileup.ReadPileup;
+import org.broadinstitute.hellbender.utils.read.ArtificialReadUtils;
+import org.broadinstitute.hellbender.utils.read.GATKRead;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.util.Arrays;
+
+import static org.testng.Assert.*;
+
+/**
+ * Created by David Benjamin on 2/15/17.
+ */
+public class GATKProtectedVariantContextUtilsUnitTest {
+    @Test
+    public void testGetPileup() {
+        final SAMFileHeader header = ArtificialReadUtils.createArtificialSamHeader(1, 1, 1000);
+        final Locatable loc = new SimpleInterval("chr1", 10, 10);
+        final int readLength = 3;
+
+        //this read doesn't overlap {@code loc}
+        final GATKRead read1 = ArtificialReadUtils.createArtificialRead(header, "read1", 0, 1, readLength);
+        read1.setBases(Utils.dupBytes((byte) 'A', readLength));
+        read1.setBaseQualities(Utils.dupBytes((byte) 30, readLength));
+        read1.setCigar("3M");
+
+        //this read overlaps {@code loc} with a Q30 'A'
+        final GATKRead read2 = ArtificialReadUtils.createArtificialRead(header, "read1", 0, 10, readLength);
```

Comment:
typo: the read names (the second argument to `createArtificialRead`) should be read2, read3, and so on.